igvm/vm.py

```diff
@@ -79,13 +80,29 @@ def __init__(self, dataset_obj, hypervisor=None):
         # directly on running VM.
         self.mounted = False
 
+        if vg_name is None:
+            # Check if we can find it
+
+            if hypervisor is None:
+                # This is probably caused by an error,
+                # but we will pass as None
```

Comment:
This is actually the case for AWS, as we don't care about the hypervisors there. 
Please amend the comment, as it's not an error per se.